.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -217,3 +223,99 @@ module upload_sfn {
   lambda_error_handler   = module.upload_error_lambda.arn
   deployment_stage       = local.deployment_stage
 }
+
+module dataset_submissions_lambda {
+  source                     = "../lambda"
+  image                      = "${local.lambda_dataset_submissions_repo}:${local.image_tag}"
+  name                       = "dataset-submissions"
+  custom_stack_name          = local.custom_stack_name
+  remote_dev_prefix          = local.remote_dev_prefix
+  deployment_stage           = local.deployment_stage
+  artifact_bucket            = local.artifact_bucket
+  cellxgene_bucket           = local.cellxgene_bucket
+  dataset_submissions_bucket = local.dataset_submissions_bucket
+  lambda_execution_role      = aws_iam_role.dataset_submissions_lambda_service_role.arn
```

Comment:
We need a different role for this lambda since it needs to invoke the ingestion step function.

Reply:
Are you saying that `dataset_submissions_lambda_service_role` will fill that need? Or that this is a to-do?

Reply:
Yup, this is that role. It has the `states:StartExecution` action granted below.